### PR TITLE
[VEN-2515] XVS Buyback and Funds Allocation

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2515

### CODE REVIEW

File: vips/vip-279/bsctestnet.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const XVS_VAULT_TREASURY = "0x317c6C4c9AA7F87170754DB08b4804dD689B68bF";
+export const VTREASURY = "0x8b293600c50d6fbdc6ed4251cc75ece29880276f";
```
Comment (Debugger022) [inline vips/vip-279/bsctestnet.ts:5]: We can remove them if they are not being used.

File: vips/vip-280/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+export const XVS_VAULT_TREASURY = "0x269ff7818DB317f60E386D2be0B259e1a324a40a";
+export const VTREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+export const XVS_VAULT = "0x051100480289e704d20e9DB4804837068f3f9204";
+
+export const XVS_FOR_XVS_STORE = parseUnits("72905.06", 18);
+export const XVS_FOR_V_TREASURY = parseUnits("21789.17", 18);
+export const XVS_DISTRIBUTION_SPEED = parseUnits("0.063888888888888888", 18); // 1,840 XVS/day
+
+export const vip280 = () => {
```
Comment (Debugger022) [inline vips/vip-280/bscmainnet.ts:15]:
```suggestion
const vip280 = () => {
```
we can remove export in all vips as they are being exported at the end.

**Review by Debugger022 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Review by kkirka — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/282?chainId=56

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/283?chainId=56

**Pull request merged**